zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -955,6 +1200,42 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/CamundaUserResponse"
+    AbstractRole:
+      type: object
+      properties:
+        name:
+          type: "string"
+        description:
+          type: "string"
```

Comment:
🔧 We can also integrate those properties directly into the `RoleRequest`, can't we? The `AbstractRole` is only used there.